src/scripts/install.py

```diff
@@ -63,6 +63,19 @@ def log_level():
 
     return (options, args)
 
+def prepend_destdir(path):
+    """
+    Needed because os.path.join() discards the first path if the
+    second one is absolute, which is usually the case here. Still, we
+    want relative paths to work and leverage the os awareness of
+    os.path.join().
+    """
+    try:
+        destdir = os.environ['DESTDIR']
+        return os.path.join(destdir, path.lstrip('/'))
+    except KeyError as e:
+        return path
+
```

Comment:
👍 

Of course, it would be good to have Wondows support. My feeling was just that Windows has no priority here because packaging usually works differently and implementing this for Unix is hard enough. In this case assertion is not needed but implementation must be good. And leftstriping of / will not be the answer